doc/source/examples/patch_1_2.nblink

```diff
@@ -0,0 +1,3 @@
+{
+  "path": "../../../examples/models/sklearn_iris/sklearn_iris.ipynb"
```

Comment:
why is this file named patch but points to sklearn_iris. Same for above but other way round?

Reply:
Nice catch it seems I ended up editing the wrong file, I've updated it now